docs/api/utils/mergeWithKey.md

```diff
@@ -1,17 +1,40 @@
 ---
-title: mergeWithKey()
+title: mergeWithKey(inputObject)
+sidebar_label: mergeWithKey()
 ---
 
 Emits the values from all the streams of the provided object, in a result
 which provides the key of the stream of that emission.
 
-### Arguments
+```ts
+function mergeWithKey<
+  O extends { [P in keyof any]: ObservableInput<any> },
+  OT extends {
+    [K in keyof O]: O[K] extends ObservableInput<infer V>
+      ? { type: K; payload: V }
+      : unknown
+  }
+>(inputObject: O, concurrent?: number, scheduler?: SchedulerLike): 
+  Observable<OT[keyof O]>
+```
+
+#### Arguments
 
 - `inputObject`: Object of streams
+- `concurrent`: (Optional) **TODO**
+- `scheduler`: (Optional) **TODO**
+
+#### Returns
+
+`Observable<OT[keyof O]>`: **TODO**
```

Comment:
This is going to be tough :joy: 

@voliva help!!!

I would say something like:

An observable that emits a flux-like object that contains 2 properties:
- `key`: they key of the stream that has emitted
- `payload`: the value emitted by that stream